src/main/scala/strawman/collection/immutable/BitSet.scala

```diff
@@ -27,6 +27,7 @@ sealed abstract class BitSet
   def empty: BitSet = BitSet.empty
 
   def iterableFactory = Set
+  def sortedIterableFactory = SortedSet
```

Comment:
Because the `SortedIterableFactory` can build a collection for every element type with an `Ordering`.